lib/ansible/modules/cloud/amazon/ec2_lc.py

```diff
@@ -83,16 +98,19 @@
     required: false
   spot_price:
     description:
-      - The spot price you are bidding. Only applies for an autoscaling group with spot instances.
+      - The spot price you are bidding. Only applies for an autoscaling group
+        with spot instances.
     required: false
   instance_monitoring:
     description:
-      - whether instances in group are launched with detailed monitoring.
+      - Whether instances in group are launched with detailed monitoring.
     default: false
+    required: false
```

Comment:
Thanks for the feedback and apologies for the delay in getting started on these. Have removed all required: false as advised.